pkg/authorization/static/static.go

```diff
@@ -60,17 +60,33 @@ func (saConfig StaticAuthorizationConfig) Matches(a authorizer.Attributes) bool
 	isAllowed := func(staticConf string, requestVal string) bool {
 		if staticConf == "" {
 			return true
-		} else {
-			return staticConf == requestVal
 		}
+		return staticConf == requestVal
+	}
+	isGroupAllowed := func(configGroups, requestGroups []string) bool {
+		if len(configGroups) == 0 {
+			return true
+		}
+		// O(n^2) is fine here as the groups are small. Optimize if n grows large.
```

Comment:
How do you plan to measure n growing large? I'd just use sets right away for the groups from config.